server/server.ts

```diff
@@ -211,6 +211,8 @@ app.use(express.json({
         message: 'Invalid digest'
       })
     }
+
+    req.rawBody = buf
```

Comment:
I suggest to add this only if `req.originalUrl` starts with `/plugins/` because we don't need it in peertube